.env

```diff
@@ -52,6 +52,7 @@ BUCKET_SECRET=minioadmin
 
 ###> symfony/mailer-bundle ###
 ADMIN_MAILS='["antoine.aresu@reconnect.fr", "celine.gacon@reconnect.fr", "nils.lapotre@reconnect.fr", "thibaut.cheymol@reconnect.fr"]'
+DUPLICATE_DEFAULT_RECIPIENT='respo@antennes.fr'
```

Comment:
REGIONAL_MANAGER_MAIL could be clearer ?

Reply:
Oh no i actually forgot to change it. It's support@reconnect.fr.